steward/tests/acceptance.rs

```diff
@@ -30,65 +35,436 @@ use steward::config::StewardConfig;
 /// invocations as `cargo test` executes tests in parallel by default.
 pub static RUNNER: Lazy<CmdRunner> = Lazy::new(|| CmdRunner::default());
 
-/// Use `StewardConfig::default()` value if no config or args
+/// Use command-line argument value
 #[test]
-#[ignore]
-fn start_no_args() {
+fn eth_keys_add() -> io::Result<()> {
+    let key_dir = TempDir::new("test_key")?;
+    let config_dir = TempDir::new("test_key")?;
+    let config_file_path = config_dir.path().join("config.toml");
+    let mut f = File::create(config_file_path.clone())?;
+    let configu = StewardConfig {
+        cellars: StewardConfig::default().cellars,
+        ethereum: StewardConfig::default().ethereum,
+        keys: KeysConfig {
+            keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+            rebalancer_key: "cellar".to_string(),
+        },
+        keystore: key_dir.path().as_os_str().to_str().unwrap().to_string(), // fix this before pushing
+        server: StewardConfig::default().server,
+        gravity: StewardConfig::default().gravity,
+        cosmos: StewardConfig::default().cosmos,
+        metrics: StewardConfig::default().metrics,
+    };
+    f.write_all(toml::to_string(&configu).unwrap().as_bytes())?;
     let mut runner = RUNNER.clone();
-    let mut cmd = runner.arg("start").capture_stdout().run();
-    cmd.stdout().expect_line("Hello, world!");
+    let cmd = runner
+        .args(&[
+            "-c",
+            config_file_path.to_str().unwrap(),
+            "keys",
+            "eth",
+            "add",
+            "sha",
+        ])
+        .capture_stdout()
+        .run();
+    // Check that command executes without error.
     cmd.wait().unwrap().expect_success();
+
+    f.sync_all()?;
+
+    Ok(())
```

Comment:
Before ending the test, it's good that we see that the command succeeded, but we should also inspect the end state to see that it did what it was expected to do. In this case, we should check for the presence of the new key file. This applies broadly to the rest of the tests as well.

Reply:
I think this commit addresses it: https://github.com/PeggyJV/steward/pull/80/commits/47129b7e331af93a8988b16e6877e95745763dac